frontend/app/src/components/settings/general/AbbreviateNumberSetting.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <settings-option
+    #default="{ error, success, update }"
+    setting="abbreviateNumber"
+    frontend-setting
+  >
+    <v-switch
+      v-model="abbreviate"
+      class="frontend-settings__fields__abbreviate_number"
```

Comment:
Do we use this class anywhere? I don't see any style so if it is only testing maybe we can replace with `data-cy`